notebooks/concepts/imviz_roman_asdf.ipynb

```diff
@@ -129,16 +152,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "plot_options = imviz.plugins['Plot Options']\n",
+    "\n",
     "for viewer_name in imviz.app.get_viewer_reference_names():\n",
     "    viewer = imviz.app.get_viewer(viewer_name)\n",
-    "\n",
-    "    # set colormap and stretch:\n",
-    "    viewer.stretch = 'log'\n",
-    "    viewer.set_colormap('Viridis')\n",
-    "    viewer.cuts = [1, 1e4]\n",
+    "    plot_options.viewer = viewer_name\n",
     "    \n",
-    "    # set zoom level\n",
-    "    viewer.zoom(1.25)"
+    "    for data in viewer.data():\n",
```

Comment:
Can't we use multi-select here?

Also, you don't need the overhead of `data()` because you can grab labels from `data_collection` directly.

Reply:
There's a different subtlety there that I'm avoiding, because we're WCS-linked and I don't want to have to filter out the WCS only layer. For this concept notebook, the `viewer.data()` approach seems ok to me.